metaflow/plugins/kfp/tests/flows/kfp_flow.py

```diff
@@ -27,22 +48,31 @@ def start(self):
         self.dividend = 26
         self.divisor = 7
         self.next(self.end)
+        env_image_tag = os.getenv("IMAGE_TAG")
+        assert not env_image_tag.endswith("_kfp_step")
+        assert_step_image(env_image_tag)
 
     @kfp(
         preceding_component=div_mod,
         preceding_component_inputs="dividend divisor",
         preceding_component_outputs="quotient remainder",
+        image=os.getenv("KFP_STEP_IMAGE"),
```

Comment:
Is this comment for this line?
> "We use os.getenv instead of environ to ensure this flow
        runs correctly on the local machine (which doesn't have the KFP_STEP_IMAGE
        and IMAGE_TAG env vars)."

Reply:
Correct, but also for line `env_image_tag = os.getenv("IMAGE_TAG")` (wherever `os.getenv` is used essentially).

Reply:
Can you please put in the  comment what is the different behavior between os.getenv and environ?

Reply:
Removed this comment entirely in favor of `image=os.getenv("KFP_STEP_IMAGE", None)`.